examples/dht_node.rs

```diff
@@ -79,7 +81,15 @@ fn main() {
     // Create a channel for server to communicate with network
     let (tx, rx) = mpsc::unbounded::<(DhtPacket, SocketAddr)>();
 
+    let tx_c = tx.clone();
+    let dht_send_fn = Arc::new(move |packet: DhtPacket, addr: SocketAddr|
```

Comment:
You could have give a copy of `tx` to NetCrypto instead of `dht_send_fn`. Why so complicated?

Reply:
c-toxcore has a lot of callbacks in `net_crypto` module, I'm not sure how we will deal with them. It was just an attempt to use callbacks instead of stream for each callback. But in this particular case you are right - it's easier to pass `tx` directly.